bgpd/bgp_evpn.c

```diff
@@ -3561,8 +3561,12 @@ static int update_advertise_vni_routes(struct bgp *bgp, struct bgpevpn *vpn)
 		const struct prefix_evpn *evp =
 			(const struct prefix_evpn *)bgp_dest_get_prefix(dest);
 
-		/* Identify MAC-IP local routes. */
-		if (evp->prefix.route_type != BGP_EVPN_MAC_IP_ROUTE)
+		/*
+		 * We have already processed type-3 routes.
+		 * Process only type-1 and type-2 routes here.
+		 */
+		if (evp->prefix.route_type != BGP_EVPN_MAC_IP_ROUTE &&
```

Comment:
nit: this needs a quick clang-format run.

Reply:
I ran checkpatch.sh and it does not complain for this diff. Do I still need to run clang-format? Isn't checkpatch.sh sufficient to ensure styling guidelines?

Reply:
you do need to run clang over this for nits ... :-)